packages/wrangler/src/__tests__/r2.test.ts

```diff
@@ -157,7 +157,7 @@ describe("r2", () => {
 				  wrangler r2 bucket info <bucket>    Get information about an R2 bucket
 				  wrangler r2 bucket delete <bucket>  Delete an R2 bucket
 				  wrangler r2 bucket sippy            Manage Sippy incremental migration on an R2 bucket
-				  wrangler r2 bucket catalog          Manage the data catalog for your R2 buckets - provides an Iceberg REST interface for query engines like Spark, DuckDB, and Trino [open-beta]
+				  wrangler r2 bucket catalog          Manage the data catalog for your R2 buckets - provides an Iceberg REST interface for query engines like Spark, and PyIceberg [open-beta]
```

Comment:
Can you remove the comma here?

Reply:
As well as other spots with similar wording

Reply:
TIL: oxford comma usage for a list of two items :P